python/ray/autoscaler/_private/aws/node_provider.py

```diff
@@ -458,6 +471,20 @@ def terminate_node(self, node_id):
         # the node cache is updated.
         pass
 
+    def check_ami_cwa_installation(self, config):
```

Comment:
You might need to put an underscore in front of this method for it to pass CI.
(If I remember right, there's a test that checks that no non-underscore methods are added to NodeProvider implementation that aren't in the NodeProviderInterfrace.)

Reply:
Thanks for the reminder, will put underscore before this function in updated commit.

Reply:
...actually, I'm getting this confused with a similar test for another class. Still good to keep consistency, though!